megamek/src/megamek/common/weapons/ADAMissileWeaponHandler.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * MegaMek - Copyright (C) 2005 Ben Mazur (bmazur@sev.org)
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+package megamek.common.weapons;
+
+import megamek.common.*;
+import megamek.common.actions.WeaponAttackAction;
+import megamek.common.options.OptionsConstants;
+import megamek.server.GameManager;
+
+import java.util.Vector;
+
+/**
+ * @author Sebastian Brocks
```

Comment:
Let's put a meaningful comment on the class.

Reply:
Done.